frontend/ui/screensaver.lua

```diff
@@ -383,7 +396,7 @@ function Screensaver:show(event, fallback_message)
         end
 
         if message_pos == "middle" or message_pos == nil then
-            widget = InfoMessage:new{
+            message_widget = InfoMessage:new{
```

Comment:
I think you'll need `local message_widget` somewhere, so it's really a local.
There was a complain about `local message_widget = nil` just because of the `= nil` assignment I guess.

Reply:
Oops, I went to delete the "= nil" and deleted the whole line.

I think it was complaining because message_widget gets assigned in both the if and else clauses so the nil value was never used? Anyway, hopefully will work now.